spyder_kernels/console/kernel.py

```diff
@@ -124,6 +125,18 @@ def set_pdb_ignore_lib(self, state):
         if self._pdb_obj:
             self._pdb_obj.pdb_ignore_lib = state
 
+    def update_syspath(self, path_dict, new_path_dict):
+        """Update the path of the kernel."""
+        # Remove old paths
+        for path in path_dict:
+            while path in sys.path:
+                sys.path.remove(path)
+
+        # Add new paths
```

Comment:
Please expand this comment to explain why we need to use `reversed` in the line below.